src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -225,6 +225,22 @@ void CompilerToVM::Data::initialize(JVMCI_TRAPS) {
 #undef SET_TRIGFUNC
 }
 
+static jboolean is_c1_supported(vmIntrinsics::ID id){
+    jboolean supported = false;
+#ifdef COMPILER1
```

Comment:
Header `c1/c1_Compiler.hpp` at line 25 was introduced in JDK-8313372.
I suggest adding `#ifdef COMPILER1` directive for this header as well.

The same to the C2 case, i.e. header `opto/c2compiler.hpp`.